rules/files-not-hash-config.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Can this be implemented in file-hash?

Reply:
Renamed to `file-hashes-not-exist` rule.
This rule is intended for global file signature searches. The use case is a bit different from `file-hash`, where it's used to detect if a certain file is changed.